app/components/Distribution.tsx

```diff
@@ -81,10 +82,13 @@ const Distribution = ({ subjectArea, catalogNumber }: DistributionProps) => {
 
   const gradeCountsForInstructorNameForTerm =
     gradeCountsByInstructorNameByTerm?.[selectedInstructorName]?.[selectedTerm];
+  const gradeCountArray = Object.values(gradeCountsForInstructorNameForTerm ?? {});
   const totalGradeCountForInstructorNameForTerm = sum(
-    Object.values(gradeCountsForInstructorNameForTerm ?? {}).map(Number),
+    gradeCountArray.map(Number),
```

Comment:
could we add `.map(Number)` to line 85 instead of here on 87? think it makes more sense for `gradeCountArray` to be an array of numbers

Reply:
Yep, makes sense. In actuality, `gradeCountArray` is already an array of numbers, so there's no need to map. Oversight on my part. Fixed.